### PR TITLE
Add our projects feature

### PR DESCRIPTION
**Pull request by GaetanNDAYAMBAJE**
Add projects page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,10 +9,12 @@
...
-          ><a href="ourmission.html">Our mission</a>
-          <a href="garley.html"></a>
+      <a href="index.html">Home</a>
+      <a href="contactus.html">Contact Us</a
+      ><a href="about.html">About us</a
+      ><a href="ourmission.html">Our mission</a>
+      <a href="garley.html">Gallery</a>
+      <a href="ourprojects.html">Our projects</a>
     </header>
     <div class="body"></div>
```
Comment (benshidanny11) [inline index.html:19]: Hello gaetan, you did a good job

File: ourprojects.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          </header>
+        <h1>Project 1</h1>
+       
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus veritatis laborum libero! Dicta expedita dolorem neque accusantium exercitationem deserunt iure eveniet tempore voluptate ut corrupti voluptatibus tenetur perferendis doloribus, omnis inventore ipsa enim! Aut placeat debitis deserunt excepturi ut laboriosam?</p>
+        
+        <br>
+        <h1>Project 2</h1>
+        
+            <p>
+             Lorem ipsum dolor sit amet consectetur adipisicing elit. Delectus, qui laborum exercitationem sit distinctio voluptas laboriosam quasi. Aspernatur explicabo ullam corporis officia, inventore veniam saepe amet, in earum ea, itaque accusantium nam assumenda illum soluta nisi maxime facere nobis provident.
```
Comment (benshidanny11) [inline ourprojects.html:30]: Good job

**Review by benshidanny11 — APPROVED** [review]
You did a good job, your project is approved

File: ourprojects.html (hunk 2 above)
Comment (GaetanNDAYAMBAJE) [reply ourprojects.html:30]: Thank you Supervisor

**Pull request merged**